compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -344,7 +344,9 @@ private String getJarFileName(ModuleContext moduleContext) {
             String documentName = moduleContext.documentContext(documentId).name();
             jarName = getFileNameWithoutExtension(documentName);
         } else {
-            jarName = moduleContext.moduleName().toString();
+            jarName = moduleContext.descriptor().org().value() + "-" +
```

Comment:
Shall we move this to a util method in `ProjectUtils`? We can call the same in BuildLangLib as well.

Reply:
Added